release-tools/SIDECAR_RELEASE_PROCESS.md

```diff
@@ -17,7 +17,7 @@ The release manager must:
 Whenever a new Kubernetes minor version is released, our kubernetes-csi CI jobs
 must be updated.
 
-[Our CI jobs](https://k8s-testgrid.appspot.com/sig-storage-csi-ci) have the
```

Comment:
pls revert change in release-tools since it requires change in csi-release-tools repo, and then make vendor change.

Reply:
pls revert all changes under `release-tools` dir, thanks.

Reply:
ok